bundles/kernel-aws

```diff
@@ -1,8 +1,8 @@
 # [TITLE]: kernel-aws
-# [DESCRIPTION]: Run the kvm specific kernel
+# [DESCRIPTION]: Run the Kernel-based Virtual Machine (KVM) kernel.
```

Comment:
@bryteise. I requested a revision to this description via email because original description is a duplicate of `kernel-kvm.`